docs/main/06-kratix-cli/00_intro.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Introduction
+description: Introduction to Kratix and how to get started with the Kratix CLI
+sidebar_label: Introduction
+id: intro
+---
+
+The Kratix CLI is a tool designed to help you build promises and manage your Kratix
+installation.
+
+## Installation
+
+To install the Kratix CLI, go to the [releases
+page](https://github.com/syntasso/kratix-cli/releases), download the latest release of
+the binary for your platform.
+
+Once you have downloaded the binary, you can install it by moving it to a directory in
+your PATH.
+
+To verify your installation, run:
+
+```shell-session
+kratix --version
+```
+
+### Enable shell autocompletion
+
+To configure your shell to load the CLI bash completions, add to your profile:
+
+```shell-session
+. <(kratix completion bash)
+```
+
+For a complete list of supported shells, run:
+
+```shell-session
+kratix completion --help
+```
+
+## Writing Promises with the Kratix CLI
+
+The CLI provides a set of commands to help you quickly iterate through the process of
+writing a Promise. It supports building promises from scratch, or initialising it from a
+Helm chart or from an existing Operator.
+
+The examples below will walk you through building a promise for each of the three
+supported scenarios.
+
+### From Scratch
+
+#### Initialising a new Promise
+
+To initialise a new Promise from scratch, run the [kratix init promise](./reference/kratix-init-promise) command:
+
+```shell-session
+kratix init promise mysql \
+  --group mygroup.org \
+  --kind Database \
+  --version v1alpha1
+```
+
+Where:
+
+- `mysql` is the name of the Promise
+- `mygroup.org` is the Group of the Promised resource
+- `Database` is the Kind provided by the Promise
+- `v1alpha1` is the Version of the Promised resource
```

Comment:
Does it make sense to say:
```
- `mygroup.org` is the Group of the Promised API resource
- `Database` is the Kind provided by the Promise
- `v1alpha1` is the Version of the Promised API resource
```

Seems that we refer to the CRD provided by Promise as "API" in the doc, so perhaps it is clearer this way?